BackEnd/server.js

```diff
@@ -1,39 +1,32 @@
 const express = require("express");
 const cors = require("cors");
-const ChatGpt = require("../FrontEnd/src/Components/CardData/ChatGpt.json");
-const Midjourney = require("../FrontEnd/src/Components/CardData/Midjourney.json");
-const DallE = require("../FrontEnd/src/Components/CardData/DallE.json");
-const StableDiffusion = require("../FrontEnd/src/Components/CardData/StableDiffusion.json");
-const Slides = require("../FrontEnd/src/Components/CardData/Slides.json");
+const ChatGpt = require("./Router/ChatGpt.router");
+const Midjourney = require("./Router/Midjourney.router");
+const DallE = require("./Router/DallE.router");
+const StableDiffusion = require("./Router/StableDiffusion.router");
 
-const ChatGptPath = "../../FrontEnd/src/Components/CardData/ChatGpt.json";
-const MidjourneyPath = "../../FrontEnd/src/Components/CardData/Midjourney.json";
-const DallEPath = "../../FrontEnd/src/Components/CardData/DallE.json";
-const StableDiffusionPath =
-  "../../FrontEnd/src/Components/CardData/StableDiffusion.json";
-const SlidesPath = require("../FrontEnd/src/Components/CardData/Slides.json");
+const connection = require("./Config/db");
```

Comment:
`db.js` file exports an object `{ connection }` which means here `connection` variable will be an `Object`, adding async/await to a object is logically incorrect. Please verify the logic added on line #26 
`await connection;`

Reply:
Made changes!